test/integration/imagechange_buildtrigger_test.go

```diff
@@ -477,12 +479,21 @@ func TestMultipleImageChangeBuildTriggers(t *testing.T) {
 		if err != nil {
 			t.Fatalf("Couldn't create Image: %v", err)
 		}
+
+		var newBuild *buildapi.Build
+		var event watchapi.Event
 		// wait for initial build event from the creation of the imagerepo
-		event := <-watch.ResultChan()
+		for {
+			event = <-watch.ResultChan()
+			newBuild = event.Object.(*buildapi.Build)
```

Comment:
potential, but unless we start seeing flakes of that nature (and to my knowledge so far we haven't), I don't think it's worth doing anything right now.